pkg/minikube/notify/notify_test.go

```diff
@@ -34,11 +35,17 @@ import (
 	"k8s.io/minikube/pkg/version"
 )
 
+func TestMaybePrintUpdateTextFromGithub(t *testing.T) {
+	if MaybePrintUpdateTextFromGithub() {
+		t.Fatal("MaybePrintUpdateTextFromGithub() expected to return false for basic setup, bot got true")
+	}
+}
+
 func TestShouldCheckURL(t *testing.T) {
 	tempDir := tests.MakeTempDir()
 	defer os.RemoveAll(tempDir)
 
-	lastUpdateCheckFilePath := filepath.Join(tempDir, "last_update_check")
+	lastUpdateCheckFilePath := path.Join(tempDir, "last_update_check")
```

Comment:
Why this change? Won't this break on Windows?

Reply:
If that's the case, it is my fault, I thought it was the other way around. Where `filepath` was the problem. Sorry @n0npax 

Thanks @tstromberg

Reply:
No problem - it is confusing in minikube, but here is the rule of thumb:

- Local filesystem path -> `filepath.Join`
- VM filesystem path -> `path.Join`
- URL: `path.Join`

Reply:
reverted to `filepath` Thanks guys!